script-library/container-features/src/devcontainer-features.json

```diff
@@ -1121,6 +1121,24 @@
 				"ruby-rails-postgres",
 				"python-3-postgres"
 			]
+		},
+		{
+			"id": "jupyterlab",
+			"name": "Jupyter Lab",
+			"documentationURL": "https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/jupyterlab.md",
+			"options": {
+				"version": {
+					"type": "string",
+					"proposals": ["latest", "3.6.2"],
+					"default": "latest",
+					"description": "Select or enter a jupyterlab version."
+				}
+			},
+			"buildArg": "_VSC_INSTALL_JUPYTERLAB",
+			"extensions": [],
+			"include": [
```

Comment:
`include` and `exclude` need a bit of work, but today this would only have this feature show up for `codespaces-linux` and none of our other base definitions.  I don't think there's any reason to be so restrictive?